Content.Server/Chat/ChatManager.cs

```diff
@@ -71,6 +76,19 @@ public void EntitySay(IEntity source, string message)
             msg.MessageWrap = $"{source.Name} says, \"{{0}}\"";
             msg.SenderEntity = source.Uid;
             _netManager.ServerSendToMany(msg, clients.ToList());
+
+            var entities = source.EntityManager.GetEntitiesInRange(pos, VoiceRange);
+            if (entities.Count() > 0)
+            {
+                foreach (var entity in entities)
+                {
+                    if (entity.TryGetComponent<ListeningComponent>(out ListeningComponent listener)
```

Comment:
Ok, I know what you were going for, having a listening component separate from the radio component (letting other things besides radios "hear")
But as it stands this component is useless because it only works with a `RadioComponent` (it does nothing itself)

What I'd do instead, is look up all components on this entity with `IListen/IListener/IListenAct` (which `RadioComponent` would then implement) and pass the message on to all of them.

just something simple like

```cs
public interface IListen
{
    void HeardSpeech(string speech, Entity speaker)
}
```
(I think passing along the speaker is good, a generic listener would need this more than radios tho tbf.)

And then here you'd have

```cs
foreach (var listener in entity.GetAllComponents<IListen>())
{
    listener.HearSpeech(message, source);
}
```

Reply:
We could possibly add back `ListenerComponent`, and a `ListenerSystem`, so we can ask for only listeners in range, rather than all entities in range and asking if any of their n components can listen. (We'd keep the `IListen` interface, but it'd be dispatched to via `ListenerComponent`)

(You can leave that as a comment above this loop if you want as a TODO though)
But for now I wouldn't worry.